src/StoreKeeper/WooCommerce/B2C/Database/DatabaseConnection.php

```diff
@@ -121,4 +122,27 @@ public function prepare(QueryInterface $query)
             $query->getBindValues()
         );
     }
+
+    public static function formatToDatabaseDate(\DateTime $dateTime): string
+    {
+        $dateTime->setTimezone(new \DateTimeZone('UTC'));
+
+        return $dateTime->format(DateTimeHelper::MYSQL_DATE_FORMAT);
+    }
+
+    public static function formatFromDatabaseDate(string $date): \DateTime
```

Comment:
```suggestion
    public static function formatFromDatabaseDateIfNotEmpty($date): ?\DateTime
    {
     if( !is_string($date) ) {
        return null;
     }
     return self::formatFromDatabaseDate($data)
     } 
     
     public static function formatFromDatabaseDate(string $date): \DateTime
```


Sometimes you pass results straight than it will fatal it not string, good to have anothed function for those cases.
see:
https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/93/files/579d21a14c80681b010d228b69f25dc9480f4aba..6ca7fbbf2cb0a6bae173d20d7fd72ea3ca8f84e8#diff-5061114bd86340850c19a04b939a8ecbde6dff97786c8b89cfd68042595b7a4eR139****

Reply:
Added this method and implemented to affected code.